deploy/config-spec/properties.yaml

```diff
@@ -6,29 +6,8 @@ spec:
         regex: "^((https?|ftp|file)://)?[-a-zA-Z0-9+&@#}/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]"
         examples:
           - "https://www.stackable.de/blog/"
-    - unit: &unitPort
-        name: "port"
-        regex: "^([0-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$"
 
 properties:
-  - property: &opaPort
```

Comment:
Do we want to remove the port from p-c? I thought this was supposed to include the kitchen sink, even if we don't support changing it.

Reply:
Now it was a required property (meaning p-c is complaining if you leave it out). Since we hard code it i think there is no need to have it in there?